app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    # add people to list
+    for dict_of_person in people:
+        new_person = Person(dict_of_person["name"], dict_of_person["age"])
+        person_list.append(new_person)
+    # add attributes to each person
+    for index1 in range(len(person_list)):
+        if "wife" in people[index1]:
+            if people[index1]["wife"] is not None:
+                for index2 in range(len(people)):
+                    if people[index1]["wife"] == people[index2]["name"]:
+                        person_list[index1].wife = person_list[index2]
+
+        elif "husband" in people[index1]:
+            if people[index1]["husband"] is not None:
+                for index2 in range(len(people)):
+                    if people[index1]["husband"] == people[index2]["name"]:
+                        person_list[index1].husband = person_list[index2]
+    return person_list
```

Comment:
So hard realization...... First of all why do you iterate through index? Also, why do you use loop into loop? 
You have Person.people dict use it and only one loop here

Reply:
I use the first loop to add attributes to each person and the second to find husband/wife for him, I don't know how to make it by one loop

Reply:
I use index because it is the same in the dict and in the list or person and it is easy to iterate so -__-

Reply:
Dont answer I got you )